src/components/NoteWhiteArea.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { note } from './NoteWhiteArea.scss';
+
+const NoteWhiteArea = ({ className, children, ...props }) =>
```

Comment:
That makes sense. I would give it a more semantic name though, like `HelpContainer` or `HintContainer`.

If it's meant to represent a particular style, I think it should enforce that style, and not allow the user to override it with `className` or `props`.